### DISCUSSION TITLE
Please better explain what "annotating_components" means?

### DISCUSSION THREAD

**Discussion by moyid** [explosion/spaCy#9292] [Help: Other Questions] [answered]
Hi,

I noticed a new required field now in configs called "annotating_components". Can you please explain what this means? What is the recommended setting for an NER training config?

**Comment by polm** [comment, marked as answer]
There is [a section in the docs](https://spacy.io/usage/training#annotating-components) that explains this.

**Reply by moyid** [reply]
Thanks, playing around with this now. It doesn't seem to work if I add "transformer" as a component though.

**Reply by svlandeg** [reply]
Could you elaborate on "doesn't seem to work"? What are you trying to do, what is your code, config & CLI commands, what is the output you expect, and what is the output you're getting?
